lang/rust/avro/Cargo.toml

```diff
@@ -69,12 +68,18 @@ strum = { default-features = false, version = "0.24.1" }
 strum_macros = { default-features = false, version = "0.24.0" }
 thiserror = { default-features = false, version = "1.0.31" }
 typed-builder = { default-features = false, version = "0.10.0" }
-uuid = { default-features = false, version = "1.1.1", features = ["serde", "std", "v4"] }
 zerocopy = { default-features = false, version = "0.6.1" }
 lazy_static = { default-features = false, version = "1.4.0" }
 log = { default-features = false, version = "0.4.17" }
 zstd = { default-features = false, version = "0.11.2+zstd.1.5.2", optional = true }
 apache-avro-derive = { default-features = false, version= "0.14.0", path = "../avro_derive", optional = true }
+uuid = { default-features = false, version = "1.1.1", features = ["std", "serde"] }
```

Comment:
The README now uses again `v4` but it is no more in the features here.

Reply:
The readme and the example are a demonstration of how to use the `apache-avro` library. The user is free to add `uuid` with a full set of features in their code, but for the `apache-avro` library itself the full set is not needed.  By bringing `v4` feature we add a dependency to OS-specific code which reduces the number of supported targets.  And doing it just for sake of code example, is not worth it, in my opinion. 

All other values in those examples are fixed, why only uuid must be random there?

Reply:
What exactly is the issue with `v4` ?
I didn't face any compilation errors with the diff at https://github.com/apache/avro/pull/1731#issuecomment-1164864306 and `cargo build --target wasm32-unknown-unknown`

Reply:
BTW do you use Avro+wasm in an open source project ?
I wanted to explore WebAssembly for a while but didn't have the chance yet.

Reply:
I agree that we can remove the dependency on `v4` feature!
Since it is used only for the docs it is not really needed.
Removing it!

Reply:
> BTW do you use Avro+wasm in an open source project ? I wanted to explore WebAssembly for a while but didn't have the chance yet.

Yes, we are data-streaming system in Rust and we use WebAssembly to provide users with dynamic modules for transforming, filtering, and aggregating data on the fly. Checkout https://www.fluvio.io/ and Smartmodules section is a good place to start playing with WebAssembly. Avro is one of the formats that users would like to use in their pipelines.